common/memcached/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 0.0.1
+version: 1.0.0
```

Comment:
you really want to bump the version... you know everyone needs to update the requirements.yaml

Reply:
reverted. bad habits .. ;)